source/php/Component/Acceptance/acceptance.blade.php

```diff
@@ -0,0 +1,45 @@
+<!-- acceptance.blade.php -->
+<div id="{{ $id }}" class="{{ $class }} js-suppressed-iframe" {!! $attribute !!}>
```

Comment:
We should probably rename js-suppressed-iframe to something more general since its not necessarily an iframe being suppressed.

Reply:
Agree.

Reply:
Named to suppressed-content